Frontend/ui-customer/src/Components/RateCard/View.purs

```diff
@@ -83,147 +92,365 @@ view push config =
            , imageWithFallback if config.nightCharges then "ny_ic_night,https://assets.juspay.in/nammayatri/images/user/ny_ic_night.png" else "ny_ic_day,https://assets.juspay.in/nammayatri/images/user/ny_ic_day.png"
            ]  
          ]
-      , linearLayout
+      ,linearLayout
         [ width MATCH_PARENT
         , height WRAP_CONTENT
         , orientation HORIZONTAL
-        , padding (Padding 20 20 20 8)
-        ][ textView
-           [ width WRAP_CONTENT
-           , height WRAP_CONTENT
-           , textSize FontSize.a_16
-           , color Color.black800
-           , text $ (getString MIN_FARE_UPTO) <> if config.nightCharges then " 🌙" else ""
-           ]
-         , textView
-           [ width WRAP_CONTENT
-           , height WRAP_CONTENT
-           , textSize FontSize.a_16
-           , color Color.black800
-           , text config.baseFare
-           , gravity RIGHT
-           , weight 1.0
-           ]
-         ]
+        ][PrestoAnim.animationSet [ if config.toggleRateCardView == ViewStart then (translateInXBackwardAnim if config.isCurrentStart == true then true else false) else (translateInXForwardAnim true) ] $
+          if config.toggleRateCardView == ViewStart then viewStart push config 
+          else if config.toggleRateCardView == ViewDriverAddition then viewDriverAddition push config 
+          else viewFareUpdate push config
+        ]     
+      ,linearLayout
+      [ width MATCH_PARENT
+      , height WRAP_CONTENT
+      , orientation VERTICAL
+      ][ textView
+        [ width MATCH_PARENT
+        , height WRAP_CONTENT
+        , color Color.blue800
+        , gravity CENTER
+        , fontStyle $ FontStyle.semiBold LanguageStyle
+        , text if config.toggleRateCardView == ViewStart then (getString GOT_IT) else (getString GO_BACK_)
+        , textSize FontSize.a_18
+        , lineHeight "20"
+        , padding (Padding 0 8 0 25)
+        , onClick push if config.toggleRateCardView == ViewStart then const Close else const GoToViewStart
+        ]
+      ]
+    ]      
+  ]
+viewStart :: forall w. (Action -> Effect Unit) -> Config -> PrestoDOM (Effect Unit) w 
```

Comment:
change the `viewStart` to `view_name <> View`  like `defaultRateCardView`. Check all the places and  change to understandable names.